test/other-issues/multiple-relations-same-column/entity/country.ts

```diff
@@ -0,0 +1,37 @@
+import {
+    Column,
+    Entity,
+    JoinColumn,
+    ManyToOne,
+    OneToMany,
+    PrimaryColumn,
+} from "../../../../src"
+import { Translation } from "./translation"
+
+@Entity("countries")
+export class Country {
+    @PrimaryColumn({ length: 2 })
+    code: string
+
+    @Column()
+    nameTranslationId: number
+
+    @ManyToOne(() => Translation, { createForeignKeyConstraints: false })
+    @JoinColumn({ name: "nameTranslationId", referencedColumnName: "id" })
+    // @ts-ignore
+    private nameTranslation: Translation
```

Comment:
Is it necessary to make the relation private?

Reply:
Basically, it's a **many-to-many** relationship, but without a join table. According to the documentation, 

> `@OneToMany` cannot exist without `@ManyToOne`. If you want to use `@OneToMany`, `@ManyToOne` is required.

 So these fields decorated `@ManyToOne` are technical and therefore private.